### PR TITLE
Fix for 9905 - Investigate negative numbers on Summary's timeSinceLastAttempt field

### LINKED ISSUE #9905: Investigate negative numbers on  Summary's timeSinceLastAttempt field.
This bug is to track why we have a few negative numbers on timeSinceLastAttempt 

union Office_Fluid_FluidRuntime_*
| where Data_eventName contains "fluid:telemetry:Summarizer:Running:Summarize_generate"
| extend WB = Data_hostScenarioName contains "Whiteboard"
| extend First = Data_summarizeCount == 1
| summarize count(), Seconds=avg(Data_timeSinceLastAttempt) / 1000
by Data_eventName, WB, Data_summarizeReason, First)

<HTML>
		

Data_eventName | WB | Data_summarizeReason | First | count_ | Seconds
-- | -- | -- | -- | -- | --
fluid:telemetry:Summarizer:Running:Summarize_generate | 0 | maxOps | 1 | 612 | <b>-27.3636258169935</b>
fluid:telemetry:Summarizer:Running:Summarize_generate | 0 | idle | 1 | 779462 | 5.71102638614839
fluid:telemetry:Summarizer:Running:Summarize_generate | 0 | lastSummary | 1 | 393 | 20.3479134860051
fluid:telemetry:Summarizer:Running:Summarize_generate | 0 | maxOps | 0 | 872 | 28.1187855504587
fluid:telemetry:Summarizer:Running:Summarize_generate | 0 | idle | 0 | 414794 | 36.5621380830002
fluid:telemetry:Summarizer:Running:Summarize_generate | 0 | lastSummary | 0 | 135 | 40.2513185185185
fluid:telemetry:Summarizer:Running:Summarize_generate | 0 | maxTime | 0 | 950839 | 254.752058535672
fluid:telemetry:Summarizer:Running:Summarize_generate | 0 | maxTime | 1 | 245274 | 162041.76329836

		</HTML>

From Vlad: 
It shows where problems are:

**maxOps, First -> getting negative**

**maxTime, first -> really big number**

All other items are Ok

Thisis a continuation from #9635

### PR DESCRIPTION
**Pull request by NicholasCouri**
Fix #9905  - Do not user server timestamp during the first summary.

### CODE REVIEW

File: packages/runtime/container-runtime/src/summarizerTypes.ts [hunk 1]
```diff
@@ -349,10 +349,10 @@ export interface ISummarizeHeuristicData {
     readonly lastSuccessfulSummary: Readonly<ISummarizeAttempt>;
 
     /**
-     * Initializes lastAttempt and lastSuccessfulAttempt based on the last summary.
+     * Updates lastAttempt and lastSuccessfulAttempt based on the last summary.
      * @param lastSummary - last ack summary
      */
-    initialize(lastSummary: ISummarizeAttempt): void;
+    updateWithPendingSummaryAckInfo(lastSummary: Partial<ISummarizeAttempt>): void;
```
Comment (agarwal-navin) [inline packages/runtime/container-runtime/src/summarizerTypes.ts:355]: nit: updateWithLastSummaryInfo?

Comment (agarwal-navin) [reply packages/runtime/container-runtime/src/summarizerTypes.ts:355]: Also, why make the param Partial? We are passing in all the required properties right.

Comment (NicholasCouri) [reply packages/runtime/container-runtime/src/summarizerTypes.ts:355]: Good catch - I was going to make it partial and we do not need it anymore.

File: packages/runtime/container-runtime/src/summaryGenerator.ts [hunk 2]
```diff
@@ -237,6 +237,8 @@ export class SummaryGenerator {
         const { refreshLatestAck, fullTree } = options;
         const logger = ChildLogger.create(this.logger, undefined, { all: summarizeProps });
 
+        // Note: timeSinceLastAttempt and timeSinceLastSummary for the
+        // first summary are basically teh time since the summarizer was loaded.
```
Comment (agarwal-navin) [inline packages/runtime/container-runtime/src/summaryGenerator.ts:241]: teh -> the

**Review by agarwal-navin — APPROVED** [review]

**Pull request merged**